index.js

```diff
@@ -106,7 +106,15 @@ var swaggerPaths = function (context, tag, resource) {
 };
 
 var swaggerOperation = function (context, pathParams, uriTemplate, action, tag) {
+    const operationIdBase = action.name.split(' ').join('-');
+
+    let operationId = `${action.name.split(' ').join('-')}-${action.method}`;
+    if (!operationIdBase) {
+        operationId = `${Object.keys(context.swagger.paths)[0].split('/').join('')}-${action.method}`;
+    }
```

Comment:
You should use `uriTemplate` instead of `context.swagger.paths`, because `context.swagger.paths` includes cumulated old paths.

Plus, I hope replacing `/` with `-` (same manner with `action.name` case).